### PR TITLE
added all locations to the selection location dropdown menu

### PR DESCRIPTION
**Pull request by DanielleKoduru**
# Short Description

Added "All Locations" to dropdown so it is ready for styling, moved button to event sidebar, created a location state, created a handle change for selecting the location, created a copy of location state.

## Rationale/purpose of the feature

For the user to have the option to select "all locations" in the select location dropdown. 

## Link to relevant User Story

[Add "all locations" to the dropdown on select location button and move to sidebar. Make a state and handle change for selecting location data and keeping a copy.](https://trello.com/c/6FJ3Ohxj/47-adding-all-to-the-dropdown-on-select-location-button)

## Checklist for author and reviewers

- [ ] Commits are well named and descriptive
- [ ] PR is focused on a single problem, is small enough and easy to review
- [ ] PR requests were sent to multiple reviewers
- [ ] PR is reviewed by more than one team member
- [ ] Code is consistently formatted
- [ ] Code is suitably commented on
- [ ] There is no duplicate code
- [ ] Code tries to follow best practices
- [ ] Decide with reviewers if this PR should have tests before merging

### CODE REVIEW

File: src/components/Calendar/Calendar.js [hunk 1]
```diff
@@ -164,6 +158,11 @@ const MyCalendar = (props) => {
     }
   };
 
+  // useEffect for select location dropdown
+  useEffect(() => {
```
Comment (petevs) [inline src/components/Calendar/Calendar.js:162]: I know we said, we would leave this in for Carlos to build off of but maybe we should comment it out for now? Just so it's not being called unnecessarily before it is built.

**Review by petevs — APPROVED** [review]
Looks great! Only little thing I see is to comment out the useEffect, so it isn't being unnecessarily called before it is built.

File: src/components/Calendar/Calendar.js (hunk 1 above)
Comment (DanielleKoduru) [reply src/components/Calendar/Calendar.js:162]: Thats a great point. I'll comment it out and push the code again

Comment (DanielleKoduru) [reply src/components/Calendar/Calendar.js]: I commented that useEffect out for Carlos to use in the future and re-pushed my code. Thank you for catching that Pete.

Comment (rmscrlos) [reply src/components/Calendar/Calendar.js]: Thanks for doing that!

**Comment by rmscrlos** [comment]
Great work! I will start working on the functionality 🙏🏽

**Review by priyanka10it — APPROVED** [review]
Looks great Danielle, thanks for the update.

File: src/components/Calendar/Calendar.js (hunk 1 above)
Comment (DanielleKoduru) [reply src/components/Calendar/Calendar.js]: No problem Carlos

**Pull request merged**